yeat/cli/config.py

```diff
@@ -8,44 +8,40 @@
 # -------------------------------------------------------------------------------------------------
 
 import json
+from pathlib import Path
 from warnings import warn
 
 
-ALGORITHMS = ("spades", "megahit", "unicycler", "flye", "canu")
-KEYS = ["algorithm", "extra_args"]
+ALGORITHMS = ["spades", "megahit", "unicycler", "flye", "canu"]
+KEYS1 = ["samples", "assemblers"]
+KEYS2 = ["label", "algorithm", "extra_args", "samples"]
```

Comment:
Small suggestion: if you define these constants as tuples instead of lists (as you did before with `ALGORITHMS`) you are making them immutable (as constants are expected to be) and protecting the data from both intentional and unintentional modification.